assets/js/index.js

```diff
@@ -1,20 +1,19 @@
-const app = require("express")()
-const http = require("http").createServer(app)
-const io = require("socket.io")(http)
-// const router = app.Router()
+const server = require('ws').Server
+const s = new server({port: 5001})
 
-io.on("connection", socket => {
-    console.log('connected')
-    socket.on("message", message => {
-        socket.broadcast.emit("message", message)
-    })
+s.on('connection', function (ws) {
```

Comment:
Something for a later implementation: can we store information on the connection and/or pass parameters to it? e.g. we later need to pass at least the chat-id to the connection to make sure, only people get notified of new messages, when they are subscribed to a chat. Similar to the topic subscriptions of mercure